script.js

```diff
@@ -138,9 +95,55 @@ class Player{
         console.log('player', this.player_score);
     }
 
-    calculate_card_total(){
+    calculate_card_total(score){
+        this.player_score = this.game.calculate_card_total(this.hand);
+
+        let player_outcome = document.querySelector("#player_outcome");
+
+        if (this.player_score > 21){
+            player_outcome.innerHTML = "Bust";
+         }
+         if (this.player_score == 21){
+            player_outcome.innerHTML = "Blackjack, you win!";
+         }
+    }
+
+
+
+}
+
+
+//game handles deck and functions: dealing, hit, stand, sorting cards, calculate card total
+class Game{
+
+    constructor(player, dealer){
+        //this.deck_cards = [2, 2, 2, 2, 3, 3, 3, 3, 4, 4, 4, 4, 5, 5, 5, 5, 6, 6, 6 ,6, 7, 7, 7, 7, 8, 8, 8, 8, 9, 9, 9, 9, 10, 10, 10, 10, 'J', 'J', 'J', 'J', 'Q', 'Q', 'Q', 'Q', 'K', 'K', 'K', 'K', 'A', 'A', 'A', 'A'];
+        this.deck_cards = [10, 'J', 9, 8];
+        this.player = player;
+        this.dealer = dealer;
+        // moved to classes - this.hand = [];
+
+    }
+
+
+    deal(){
 
-        this.player_score = 0;
+
+      //let chosen_card_index = getRandomArbitrary(0,this.deck_cards.length);
+      let chosen_card_index = 0;
+      let chosen_card = this.deck_cards[chosen_card_index];
+
+      let before = this.deck_cards.slice(0, chosen_card_index);
+      let after = this.deck_cards.slice(chosen_card_index + 1);
+
+      this.deck_cards = before.concat(after);
+
+      return chosen_card
+    }
+
+    calculate_card_total(hand){
+        this.score = 0;
+        this.hand = [];
```

Comment:
`this.hand` and `hand` are two different variables

`hand` is an argument coming from L144. it contains a list of either the Player's cards (L99), or the Dealer's cards (L70), depending on who's calling `calculate_card_total`.

`this.hand` is a variable you initialized, bound to `Game`, and stored an empty list into on L146.

Reply:
I understand that in concept, I'm struggling on how I access the cards that are being passed in from either Dealer or Player...

Reply:
I think I have figured it out... stay tuned (maybe)